src/pages/index.js

```diff
@@ -170,8 +170,8 @@ function handleAddCardSubmit(data) {
     .then((res) => {
       const card = createCard(res.data);
```

Comment:
` createCard(res.data);`

* I can't add new cards. See the errors https://disk.yandex.ru/i/JKE-9uRg_s5Mqg 


You need to use only `res` (the api has been fixed recently)


```js

    .then((res) => {
      const card = createCard(res);
```

Reply:
` createCard(res.data);`

* I can't add new cards. See the errors https://disk.yandex.ru/i/JKE-9uRg_s5Mqg 


You need to use only `res` (the api has been fixed recently)


```js

    .then((res) => {
      const card = createCard(res);
```